spyder_kernels/ipdb/tests/test_ipdb_kernel.py

```diff
@@ -231,6 +231,13 @@ def test_ls_path_complete(ipdb_kernel):
         assert comp['matches'] == ['"ipython\\"'], comp
 
 
+@pytest.mark.skipif(os.name == 'nt', reason="Only for linux")
```

Comment:
Please change this to

    not sys.platform.startswith('linux'), reason="Only works on Linux"

because I don't know if `/tmp` is present in macOS